src/node_crypto.cc

```diff
@@ -2431,9 +2431,8 @@ void SSLWrap<Base>::GetEphemeralKeyInfo(
             .FromJust();
         break;
       case EVP_PKEY_EC:
-      // TODO(shigeki) Change this to EVP_PKEY_X25519 and add EVP_PKEY_X448
-      // after upgrading to 1.1.1.
-      case NID_X25519:
+      case EVP_PKEY_X25519:
+      case EVP_PKEY_X448:
```

Comment:
Is a test showing that both x25519 and now x448 keys are now handled possible?

Reply:
```diff
diff --git a/test/parallel/test-tls-client-getephemeralkeyinfo.js b/test/parallel/test-tls-client-getephemeralkeyinfo.js
index 113b452db6..599306993e 100644
--- a/test/parallel/test-tls-client-getephemeralkeyinfo.js
+++ b/test/parallel/test-tls-client-getephemeralkeyinfo.js
@@ -60,3 +60,4 @@ test(2048, 'DH', undefined, 'DHE-RSA-AES128-GCM-SHA256');
 test(256, 'ECDH', 'prime256v1', 'ECDHE-RSA-AES128-GCM-SHA256');
 test(521, 'ECDH', 'secp521r1', 'ECDHE-RSA-AES128-GCM-SHA256');
 test(253, 'ECDH', 'X25519', 'ECDHE-RSA-AES128-GCM-SHA256');
+test(448, 'ECDH', 'X448', 'ECDHE-RSA-AES128-GCM-SHA256');

Reply:
Thanks, Ben. That's very helpful. I've updated the PR with that change.